src/app/features/test-records/components/base-test-record/base-test-record.component.ts

```diff
@@ -27,6 +27,7 @@ export class BaseTestRecordComponent implements AfterViewInit {
   @Input() testResult!: TestResultModel;
   @Input() isEditing: boolean = false;
   @Input() expandSections = false;
+  @Input() isContingency = false;
```

Comment:
Could we show this same review page for Test Result amendment as well?  (i.e. not just Contingency tests?)